src/utils/index.js

```diff
@@ -46,34 +46,18 @@ module.exports = {
     })
   },
 
-  httpRequest: (url, method, payload) => {
-    return new Promise((resolve, reject) => {
-      const request = new XMLHttpRequest()
-      request.onreadystatechange = () => {
-        if (request.readyState === 4 && request.timeout !== 1) {
-          if (request.status !== 200) {
-            reject(request.responseText)
-          } else {
-            try {
-              resolve(JSON.parse(request.responseText))
-            } catch (jsonError) {
-              reject(new Error(`[threeBox] while parsing data: '${String(request.responseText)}', error: ${String(jsonError)}`))
-            }
-          }
-        }
-      }
-      request.open(method, url)
-      request.setRequestHeader('accept', 'application/json')
-      // request.setRequestHeader('accept', '*/*')
-      if (method === 'POST') {
-        request.setRequestHeader('Content-Type', `application/json`)
-        request.send(JSON.stringify(payload))
-      } else {
-        request.setRequestHeader('Authorization', `Bearer ${payload}`)
```

Comment:
Token not passed in header anymore?

Reply:
Yeah, I'm not sure why we did that. Don't think the address-server checks that, but we can double check.